resources/skins.snapwikiskin.search/skins.snapwikiskin.search.js

```diff
@@ -0,0 +1,48 @@
+/** @module search */
+var Vue = require("vue").default || require("vue"),
+  App = require("./App.vue"),
+  config = require("./config.json");
```

Comment:
I cannot see config.json anywhere.

Reply:
Yeah, but I think mw and RL do some magic

Reply:
True, this was added to Vector in [this commit](https://github.com/wikimedia/Vector/commit/e0c47dc462b87fc1250d6e2dd6ab7020bced0c9a).